pwa-devdocs/src/technologies/basic-concepts/index.md

```diff
@@ -2,85 +2,85 @@
-<sub>[ *[Help write this topic][Loading and offline states]* ]</sub>
+Loading and offline are both states that must be handled by progressive web applications.  
+<sub>[ _[Help write this topic][loading and offline states]_ ]</sub>
 
 ## Container extensibility
 
-Writing extensible containers allow others to re-use and alter your container without modifying the source. 
+Writing extensible containers allow others to re-use and alter your container without modifying the source.
 
-For more information, see [Container extensibility].
+For more information, see [Container extensibility][].
 
 ## Performance patterns
 
 Performance is an important feature for a progressive web app.
-There are many strategies and patterns available to help boost the performance of a PWA.   
-<sub>[ *[Help write this topic][Performance patterns]* ]</sub>
+There are many strategies and patterns available to help boost the performance of a PWA.  
+<sub>[ _[Help write this topic][performance patterns]_ ]</sub>
 
 ## Root components and routing
 
 The root component of an application is the DOM node under which all other nodes are managed by React.
-Routing is the ability to map a URL pattern to the appropriate handler.   
-<sub>[ *[Help write this topic][Performance patterns]* ]</sub>
+Routing is the ability to map a URL pattern to the appropriate handler.  
+<sub>[ _[Help write this topic][root components and routing]_ ]</sub>
 
 ## Critical path
 
 The critical path for rendering refers to the steps the browser takes to process the HTML, CSS, and JavaScript files to display a website.
-Optimizing the critical path is important to get the best performance out of a progressive web application.   
-<sub>[ *[Help write this topic][Critical path]* ]</sub>
-
-[Redux]: https://redux.js.org/introduction/core-concepts
-
-[Service worker]: {{ site.data.vars.repo }}/issues/14
-[Component data binding]: {{ site.data.vars.repo }}/issues/9
-[Client state, reducers, and actions]: {{ site.data.vars.repo }}/issues/12
-[Loading and offline states]: {{ site.data.vars.repo }}/issues/13
-[Container extensibility]: {{ site.baseurl }}{%link technologies/basic-concepts/container-extensibility/index.md %}
-[Performance patterns]: {{ site.data.vars.repo }}/issues/16
-[Root components and routing]: {{ site.data.vars.repo }}/issues/17
-[Critical path]: {{ site.data.vars.repo }}/issues/18
-[GraphQL]: {{ site.baseurl}}{%link technologies/basic-concepts/graphql/index.md %}
-[CSS modules]: {{ site.baseurl }}{%link technologies/basic-concepts/css-modules/index.md %}
-[Application shell]: {{site.baseurl}}{%link technologies/basic-concepts/app-shell/index.md %}
+Optimizing the critical path is important to get the best performance out of a progressive web application.  
+<sub>[ _[Help write this topic][critical path]_ ]</sub>
+
+[container extensibility]: {{ site.baseurl }}{%link technologies/basic-concepts/container-extensibility/index.md %}
```

Comment:
This also doesn't render well, but I'm suspecting is just a markdown preview issue. Help me with my local, and I presume this is good to go.